ruma-appservice-api/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # [unreleased]
 
+Bug fixes:
```

Comment:
Since this changes the public API, let's declare it a breaking change instead.

Reply:
I was expecting this one ;)